test/integration/targets/postgresql/tasks/main.yml

```diff
@@ -238,6 +238,10 @@
     encryption_values: '{{ encryption_values }} + ["no"]'
   when: postgres_version_resp.stdout is version('10', '<=')
 
+# scram-sha-256 password encryption type is supported from PostgreSQL 10.0
```

Comment:
I think better to remove it
you can use intro test_password.yml the condition like this
```
when: postgres_version_resp.stdout is version('10', '>=')
```

Reply:
or define this variable into there

Reply:
better to keep main.yml cleaner (better to use it only for includes)

Reply:
agreed